subsys/bluetooth/services/ots/ots.c

```diff
@@ -385,6 +433,9 @@ int bt_ots_init(struct bt_ots *ots,
 
 	__ASSERT(ots_init->cb->obj_created,
 		 "Callback for object creation is not set");
+	__ASSERT(ots_init->cb->obj_deleted ||
+		 !BT_OTS_OACP_GET_FEAT_CREATE(ots_init->features.oacp),
+		 "Callback for object deletion is not set and object creation is enabled");
```

Comment:
Is it a requirement to support deletion if creation is supported?

Reply:
I do not believe there is a requirement for this in the OTS specification.